src/OrchardCore.Modules/OrchardCore.Admin/Startup.cs

```diff
@@ -81,4 +85,15 @@ public override void ConfigureServices(IServiceCollection services)
             });
         }
     }
+
```

Comment:
We're going to have to move al to a new project `OrchardCore.Settings.Core` because the modules shouldn't be referencing `OrchardCore.Settings`. Not sure why the `Admin` module has that reference already, we should fix that too :)